index.js

```diff
@@ -78,11 +78,9 @@ function writeFileSync (file, obj, options = {}) {
   return fs.writeFileSync(file, str, options)
 }
 
-const jsonfile = {
+module.exports = {
```

Comment:
Let's add a comment here so this doesn't get accidentally broken in the future
```suggestion
// NOTE: do not change this export format; required for ESM compat
// see https://github.com/jprichardson/node-jsonfile/pull/162 for details
module.exports = {
```